openslides_backend/shared/schema.py

```diff
@@ -25,3 +25,5 @@
 }
 id_list_schema: Schema = {**base_list_schema, "items": required_id_schema}
 fqid_list_schema: Schema = {**base_list_schema, "items": required_fqid_schema}
+
+decimal_schema: Schema = {"type": "string", "pattern": r"^-?(\d|[1-9]\d+)\.\d{6}$"}
```

Comment:
Seems like the comment here wasn't saved :( Well anyway, please use this schema also in the `fields.py` for the `DecimalField`

Reply:
Done.